src/sections/Resources/Resources-grid/filters.js

```diff
@@ -74,132 +77,160 @@ const Navigation = (props) => {
-        <div className="list">
-          <ul className={`ul ${expandType ? "ul-open" : ""}`}>
-            {typeOptions[0].subdata.map((x) => (
-              <li key={x.id}>
-                <label>
-                  <input type="checkbox"  value={x.value} onChange={props.handleChange} />
-                  <span> {x.label}</span>
-                  { types && types.map(type => (
-                    type.fieldValue === x.value ? <span key={x.id} className="total">({type.totalCount})</span> : ""
-                  ))}
-                  { typesFieldValues && !typesFieldValues.includes(x.value) ? <span key={x.id} className="total">(0)</span> : ""}
-                </label> 
-              </li>
-            ))}
-          </ul>
-        </div>
-
-        <div className="toggle-btn" onClick={function () {
-          setExpandProduct(!expandProduct); 
-        }}>
-          <p><strong>Product</strong></p>
-          {expandProduct ? (
-            <HiOutlineChevronUp
-              className="menu-icon"
-            />
-          ) : (
-            <HiOutlineChevronDown
-              className="menu-icon"
-            />
-          )}
-        </div>
-        <div className="list">
-          <ul className={`ul ${expandProduct ? "ul-open" : ""}`}>
-            {productOptions[0].subdata.map((x) => (
-              <li key={x.id}>
-                <label>
-                  <input type="checkbox"  value={x.value} onChange={props.handleChange} />
-                  <span> {x.label}</span>
-                  { products && products.map(product => (
-                    product.fieldValue === x.value ? <span key={x.id}  className="total">({product.totalCount})</span> : ""
-                  ))}
-                  { productsFieldValues && !productsFieldValues.includes(x.value) ? <span key={x.id} className="total">(0)</span> : ""}
-                </label> 
-              </li>
-            ))}
-          </ul>
+        <div className="heading">
+          <div onClick={() => {
+            setExpandFilter(!expandFilter); 
+          }} className="filter-menu-icon">
+            {expandFilter ? (
+              <button className="filter-btn">
+                <IoClose className="filter-icon filter-close" />
+                <span>Filters</span>
+              </button> 
+            )  :
+              (
+                <button className="filter-btn">
+                  <FiFilter className="filter-icon filter-open"/>
+                  <span>Filters</span>          
+                </button> 
+              )}
+          </div>
+          <p className="heading-name" >
+            <strong>Filters</strong> 
+            <span className= {`${props.resources.length === 0 ? "clear-disabled" : "clear-enabled"} desk-view`} onClick={props.clear}>
+              Clear Filters 
+            </span>
+            {expandFilter ? (<span className= {`${props.resources.length === 0 ? "clear-disabled" : "clear-enabled mob-view"}`} onClick={props.clear}>
```

Comment:
Since the only difference is the `className`, we shouldn't need to have `Clear Filters` defined twice.

Reply:
```suggestion
            {expandFilter ? (<span className= {
            `${props.resources.length === 0 ? "clear-disabled" : 
            expandFilter ? "clear-enabled mob-view" : "clear-enabled desktop-view"}
            `} 
            onClick={props.clear}>
```
Something like this?

Reply:
Putting them together won't work unfortunately
The "desk-view" is put separately and doesn't depend on the resources length where as the "mob-view" does
Tried many times but this is the only way I could make it work as of now